src/hooks/useAsync.ts

```diff
@@ -1,6 +1,6 @@
 import { useState, useCallback, useEffect } from "react"
 
-const useAsync = (asyncFuntion: (...data: any) => any) => {
+const useAsync = (asyncFuntion: (...data: any) => any, isRun = false) => {
```

Comment:
```suggestion
const useAsync = (asyncFuntion: (...data: any) => any, immediate = false) => {
```
isRun cũng sai ngữ pháp rồi